src/achievement.cpp

```diff
@@ -65,19 +66,19 @@ bool SteamManager::setAchievement(int achID)
     if (this->isInitialized)
     {
         if (trace_all || trace_achievement)
-            ffnx_trace("%s - Achievement %s set, Store request sent to Steam\n", __func__, this->achievementList[achID].chAchID);
+            ffnx_trace("%s - Achievement %s set, Store request sent to Steam\n", __func__, this->getStringAchievementID(achID));
 
-        // TODO reactivate when done all;
-        
-        // RELEASE PHASE OF STEAM ACHIEVEMENTS
-        //SteamUserStats()->SetAchievement(this->achievementList[achID].chAchID);
-        // bool success = SteamUserStats()->StoreStats();
-        //return success;
-
-        // TESTING PHASE OF STEAM ACHIEVEMENTS
         this->achievementList[achID].isAchieved = true;
-        bool success = SteamUserStats()->IndicateAchievementProgress(this->achievementList[achID].chAchID, 99, 100);
-        return success;
+        return SteamUserStats()->IndicateAchievementProgress(this->getStringAchievementID(achID), 99, 100);
+        // if(steam_achievements_debug_mode)
+        // {
+        //     return SteamUserStats()->IndicateAchievementProgress(this->getStringAchievementID(achID), 99, 100);
+        // } 
+        // else
+        // {
+        //     SteamUserStats()->SetAchievement(this->getStringAchievementID(achID));
+        //     return SteamUserStats()->StoreStats();
+        // }

```

Comment:
Better cleanup this, if you need that piece of code you can look up at the git history of this file :) Thanks

//EDIT: I also was wondering, if the flag still makes sense as this code is commented.

Reply:
I know, then I will remove the flag and the comment for now.